Order.Management/Shape.cs

```diff
@@ -7,11 +7,17 @@ namespace Order.Management
     abstract class Shape
     {
         public string Name { get; set; }
+
         public int Price { get; set; }
+
         public int AdditionalCharge { get; set; }
+
         public int NumberOfRedShape { get; set; }
+
         public int NumberOfBlueShape { get; set; }
+
         public int NumberOfYellowShape { get; set; }
+
         public int TotalQuantityOfShape()
```

Comment:
This method call can be converted to Property since method is simple 
Kindly see attached sample implementation of Shapes.cs:

[Shape.cs.txt](https://github.com/SerkoNZ/OrderManagement/files/5962400/Shape.cs.txt)